core/src/main/scala/caliban/GraphQLResponse.scala

```diff
@@ -67,3 +71,45 @@ private object GraphQLResponsePlayJson {
     }
 
 }
+
+private object GraphQLResponseZioJson {
```

Comment:
How about moving part of this code to the interop folder? (same for circe and play, as well as code in `Value` and `GraphQLRequest`) This is becoming quite bloated here.

Reply:
Sure, I can do that. That will be a breaking change though right? Since it will no longer be available as part of the companion object?

Reply:
Why wouldn't it be available? We can keep the `implicit def` in the companion object and move the implementation, no?

Reply:
Doesn't `private object` need to be in the same file?

Reply:
Does it need to be in a private object there? If yes we can still have the code inside that object call a function in interop I think?